earthfile2llb/converter.go

```diff
@@ -614,7 +614,7 @@ func (c *Converter) internalRun(ctx context.Context, args []string, secretKeyVal
 			continue
 		}
 		if ba.IsConstant() {
-			extraEnvVars = append(extraEnvVars, fmt.Sprintf("%s=\"%s\"", buildArgName, ba.ConstantValue()))
+			extraEnvVars = append(extraEnvVars, fmt.Sprintf("%s=%s", buildArgName, ba.ConstantValue()))
```

Comment:
This change isn't right probably, in case the build arg value contains a space.

Reply:
But the ConstantValue will already include either ' or " due to the ConstantValue being an Atom, so if it has a space, it doesn't matter?

Reply:
Oh... does it? If that's the case, you may be right